### ISSUE TITLE
Bug: Error when building appwrite using build instructions

### ISSUE THREAD

**Issue by ShreyasJejurkar** [appwrite/appwrite#5733]
### 💭 Description

I am trying to build appwrite repo using codespaces and following the instructions mentioned here at https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md#setup-from-source. 

But when I am running `docker compose build` from the root folder I get the below error. It looks build is getting failed when building the console project from the app folder! I can see the error for 1 second before this error is shown, but it gets cleared and I just see this red warning. 
Even more, after this error come, the build process does not terminate, it still running without any further progress. 

![image](https://github.com/appwrite/appwrite/assets/17148381/a5853ebc-9a37-4aec-ac95-c0036798d61a)

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/.github/blob/main/CODE_OF_CONDUCT.md)

**Comment by Haimantika** [comment]
Thanks for raising this issue @ShreyasJejurkar, it is working fine on my end. Our team is checking internally to see if there is any version mismatch that might have caused it.

**Comment by ShreyasJejurkar** [comment]
@Haimantika sure! Looking forward for it.

I face this issue on my new clone of the repo and following those instruction, maybe give a try with fresh clone, you might face the same issue.

**Comment by Haimantika** [comment]
> @Haimantika sure! Looking forward for it.
> 
> I face this issue on my new clone of the repo and following those instruction, maybe give a try with fresh clone, you might face the same issue.

Hi @ShreyasJejurkar are you still facing this issue? I tried with a fresh clone, and it works fine for me.

**Comment by stnguyen90** [comment]
Closing due to inactivity

**Comment by amanintech** [comment]
## TL;DR
I had faced the exact same issue and figured out a solution. This might also be a reason why the build fails.

## Solution 
1. Use Yarn instead of NPM 
2. use command `docker-compose build --no-cache` to prevent it from using any existing cache
